_includes/footer.html

```diff
@@ -97,17 +97,17 @@ <h6 class="title">Latest Updates</h6>
           </li>
           <li>
             <a href="https://gitter.im/fossasia/fossasia" target="_blank" rel="noopener noreferrer">
-              <i class="fab fa-gitter"></i>
+              <i class="fa fa-comment"></i>
```

Comment:
Why did you remove this ?

Reply:
Why has Gitter been removed?

Reply:
I have used fontawesome4.7 and it didn't have gitter icon. The link is working only the icon has been changed. I did this because fontawesome5 was giving garbage icons when used internally.

Reply:
But Gitter is required

Reply:
Please use frontawesome 5 only.

Reply:
The link to gitter should have the gitter icon